DTIPrep/DTIPrep_pipeline.pl

```diff
@@ -728,11 +725,12 @@ =head3 checkPreprocessOutputs($dti_file, $DTIrefs, $QCoutdir, ...)
 INPUTS:
   - $dti_file       : raw DWI file that was processed
   - $DTIrefs        : hash containing output names
-  - $QCoutdir       : preprocessing output directory
+  - $QCoutdir       : pre-processing output directory
   - $DTIPrepProtocol: DTIPrep XML protocol that was used to run DTIPrep
 
-RETURNS: 1 if all output files were found, undef if at least one output file
-is missing
+RETURNS:
+  - undef if at least one output file is missing
+  - 1 if all output files were found

```

Comment:
not sure why these are changed to a list? it is 2 cases of the same return, non?

Reply:
done